apps/nameguard.io/src/components/organisms/Header.tsx

```diff
@@ -34,33 +35,32 @@ export const Header = () => {
         <div className="flex flex-row items-center justify-between md:gap-5 h-[40px]">
           <div className="hidden items-center justify-center lg:flex gap-1">
             <a
-              className="px-[10px] py-[9px] not-italic text-black font-medium text-sm leading-5 rounded-md sm:hover:bg-gray-100"
               href="https://api.nameguard.io/docs"
               target="_blank"
               rel="noopener noreferrer"
             >
-              Docs
+              <Button variant="ghost" size="small">
```

Comment:
We need to align the sizes of all the buttons up here in the header. Right now the "Discuss an Integration" button is taller than the hover state of the "ghost" buttons to the left of it.

The hover state of all these "ghost" buttons should show a click area the same height as the "Discuss an Integration" button.